lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -887,6 +895,14 @@ def authorization_from(response, action, amount, options)
         [options[:order_id], response[:requestID], response[:requestToken], action, amount,
          options[:currency], response[:subscriptionID]].join(';')
       end
+
+      def in_fraud_review?(response)
+        response[:decision] == @@decision_codes[:review] && response[:ccAuthReply_reasonCode] == '100'
```

Comment:
What does `100` means? Does it mean that the review was successful?

Reply:
It means the credit card was authorized. The decision = REVIEW means that although it has been authorized, CyberSource has flagged the payment for fraud review based on its Decision Manager rules.

Reply:
What happens if review doesn't happen? What I'm getting at is that most system that rely on ActiveMerchant are not ready to handle a fraud_review state. Like nothing will happened down the line. Does this mean that the payment can be safely captured later on when the decision = REVIEW and code "100"?

Reply:
You raise a good point.  I believe payment can still be successfully captured (I've tested manually settling a transaction that is "in review" in Cybersource, and it still allows me to settle the payment).  

Cybersource flags the request for review, but my tests, there is nothing stopping someone from capturing the payment.  The credit card has been authorized and payment should be able to be captured.  I think this makes sense.

Reply:
@greg-burgoon based on my understanding of this flag, wouldn't it be better to fail the authorization and the merchant can take action to perform the review/ capture the payment based on the returned `fraud_review` value? It seems more risky to mark this as successful, when Cybersource did not ACCEPT the charge. I echo @pi3r's concern about downstream actions not set up to consider the fraud_review state at this time, and hiding fraud risk payments in success state.

Just because it allows you to settle the payment in REVIEW, I'm not convinced a user would always want to.  How are you testing capturing an in-review payment? Is that against a sandbox or production endpoint? I wonder if the behavior is different between them.